File: web2-Project/src/app/entities/avio-sediste/avio-sediste.ts
export class AvioSediste {
    id: number;
    reserved: boolean;
    isFastReservation: boolean;

    //* admin moze da posatavlja, tj. da deaktivira sediste
    isDisabled: boolean;

    deleted: boolean;

    constructor(id: number, reserved: boolean, isFastReservation: boolean) {
        this.id = id;
        this.reserved = reserved;
        this.isFastReservation = isFastReservation;

        this.isDisabled = false;
        this.deleted = false;
    }
}

File: MAANPP20/MAANPP20/Common/Models/Flights/AvioSediste.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Common.Models.Flights
{
    public class AvioSediste
    {
        [Key]
        public int id { get; set; }
        public bool reserved { get; set; }
        public bool isFastReservation { get; set; }
        public bool isDisabled { get; set; }

        public bool deleted { get; set; }
    }
}
